### DISCUSSION TITLE
We Want to Hear About Your Experience with BifroMQ!

### DISCUSSION THREAD

**Discussion by popduke** [apache/bifromq#83] [General]
Hello everyone,

Since BifroMQ went open source in July 2023, we have seen a lot of positive feedback from the community, which has been incredibly valuable to us. To better understand how you are using BifroMQ and to further improve our project, we would like to collect some information about your usage and scenarios.

If you could spare a few minutes to answer the following questions, we would greatly appreciate it:

1. How did you come across BifroMQ?
2. In what kind of projects/products are you using BifroMQ?
3. What specific problems have you solved using BifroMQ?
4. What challenges have you faced while using BifroMQ?
5. Do you have any suggestions for improvements or requests for new features in BifroMQ?

Every piece of feedback is a motivation for us to improve. We look forward to your valuable opinions!

Thank you for your support and participation!

***The BifroMQ Team***

**Comment by haroldyong** [comment]
up

**Comment by AnWen238** [comment]
[https://blog.iotcloudplatform.com/sitemap.txt](https://blog.iotcloudplatform.com/sitemap.txt)

**Comment by AnWen238** [comment]
What is the IoT cloud platform?

IOT Cloud Platform (blog.iotcloudplatform.com) focuses on IoT design, IoT programming, security IoT, industrial IoT, military IoT, best IoT projects, IoT creativity, IoT companies, Chinese IoT companies, American IoT companies, top IoT companies, IoT modules, embedded development, IoT circuit boards, IoT solutions, Raspberry Pi development and design, Arduino programming, programming languages, RFID, lora devices, IoT systems, sensors, temperature and humidity sensors, liquid level sensors, sensor devices, artificial intelligence, blockchain, robotic arms, smart homes, smart cities, smart agricultural factories, edge computing, big data, cloud computing, brain-computer interfaces, machine learning, robots, VR/AR, AI simulation technology, motion control, new energy, photovoltaic solar energy, lithium batteries, silicon brain SBB, unmanned aerospace navigation, unmanned driving, AGI, chips, semiconductors, smart hardware and other scientific and technological knowledge. https://blog.iotcloudplatform.com/
https://blog.iotcloudplatform.com/sitemap.txt
